docusaurus.config.js

```diff
@@ -26,26 +38,14 @@ export default async function createConfigAsync() {
   const docs_platform_enterprise = [
     "@docusaurus/plugin-content-docs",
     {
-      id: "platform-enterprise",
-      routeBasePath: "/platform-enterprise",
-      includeCurrentVersion: false,
-      remarkPlugins: [
-        (await import("remark-code-import")).default,
-        (await require("remark-math")).default,
-        (await import("docusaurus-remark-plugin-tab-blocks")).default,
-        (await require("remark-yaml-to-table")).default,
-      ],
-      rehypePlugins: [(await require("rehype-katex")).default],
-      editUrl: "https://github.com/seqeralabs/docs/tree/master/",
-      sidebarPath: false,
-      versions: {
-        // Replace /platform-enterprise with /platform-enterprise/24.2, when no version is specified in the URL.
-        // (Applies to latest version only)
-        [platform_enterprise_latest_version]: {
-          label: platform_enterprise_latest_version,
-          path: platform_enterprise_latest_version,
-        },
-      },
+        id: 'platform-enterprise',
+        path: 'platform-enterprise',
+        routeBasePath: 'platform-enterprise',
```

Comment:
Really think you need to include the `/` for these as it's causing strange paths.